internal/utils/utils.go

```diff
@@ -0,0 +1,18 @@
+package utils
```

Comment:
Can we avoid a utils package?

If there's only one caller of this function just leave it with the caller.
If it needs to be shared we can break it out later.